docs/learn/learn-agile-coretime.md

```diff
@@ -0,0 +1,132 @@
+one month. Heavy duty parachains which need to author a block every 12 
+seconds (or every 6s through [Async Backing](./learn-async-backing.md)), can seamlessly “renew” the [core](#core) each month. 
+Coretime renewal orders take precedence over new orders and provide protection against price 
+fluctuations, allowing parachains to plan their project budget and costs more effectively.
+
+The purchased coretime can also be split up and sold in parts, down to a single 
+block per month, allowing for secondary markets to thrive and improve the allocation efficiency
+of coretime. Furthermore, Agile Coretime offers [on-demand coretime](#on-demand-coretime) 
+functionality that enables the authoring of a parachain block on-demand.
+
+![core-usage-agile-rangeSplit](../assets/core-usage-agile-rangeSplit.png)
+
+:::info Agile Coretime is under active development
+
+The progress of Agile Coretime development can be tracked [here.](https://github.com/orgs/paritytech/projects/119/views/20)
+
+:::
+
+
+## Agile Coretime Terminology
+
+### Core
+
+The term "Core" captures the virtual abstraction of computational resources provided by 
+the relay chain to secure the blocks of a parachain, which involves a randomized set of the
+relay chain validators.
+
+### Coretime
+
+The time allocated for utilizing a core, measured in relay chain blocks. 
+
+### Bulk Coretime
+
+A fixed duration of continuous coretime represented by an NFT that can be split, 
+shared, or resold. Currently, the duration of bulk coretime is set to 28 days.
+
+### Region
+
+The NFT representing a single unit of bulk coretime.
+
+### On-demand Coretime
+
+Previously known as instantaneous coretime, the on-demand coretime refers to coretime acquired 
+through bidding in near real-time for the validation of a single parachain block on one of the 
+cores reserved specifically for on-demand orders.
+
+### On-demand Coretime Pool
+
+Set of cores that are available on-demand. Cores reserved through bulk coretime could also be
+made available in the on-demand coretime pool, in parts or in entirety.
+
+### Coretime Chain
+
+A system parachain that is responsible for the sale, manipulation of bulk coretime and 
+eventually the purchase of on-demand coretime credits. It is also responsible for 
+scheduling the respective cores on the relay chain to the parachains.
+
+### Timeslice
+
+A timeslice represents the granularity the Coretime Chain schedules cores on the Relay Chain. 
+The Coretime Chain announces a schedule for each core, comprising the next 80 relay 
+chain blocks in advance. The region length of 28 days corresponds to 5,040 timeslices.
+
+### Task
+
+The term "Task" captures the abstraction of utility of the core. Tasks on the cores
+are not just limited to securing parachains.
+
+## Agile Coretime Implementation
+
+Coretime is managed through the [broker pallet](https://github.com/paritytech/polkadot-sdk/blob/eefd5fe4499515da66d088505093f75b5aa22550/substrate/frame/broker/src/lib.rs) 
```

Comment:
Interesting, so the coretime chain will support other things than just coretime brokerage? Is that why they did not call it coretime broker chain?

Reply:
Of course. Other functionalities are planned, like adding support for coretime credits, etc., which could be part of a broker pallet or some other pallet. 

"why they did not call it coretime broker chain?"
This is partly answered in the "Product" section of FAQ uploaded. Also it is a lengthy name to have and some sort of branding decision for the system chain.